src/Symfony/Component/BrowserKit/HttpBrowser.php

```diff
@@ -39,10 +39,14 @@ public function __construct(HttpClientInterface $client = null, History $history
         parent::__construct([], $history, $cookieJar);
     }
 
+    /**
+     * @param Request $request
+     * @return Response
```

Comment:
Please remove the added annotations, we don't add them when they don't provide much.

Reply:
Got it, already saw code Coding Standard report.